extensions/amp-ad-exit/0.1/config.js

```diff
@@ -138,3 +148,29 @@ function assertTarget(name, target, config) {
     }
   }
 }
+
+/**
+ * Checks whether a vendor is valid (i.e. listed in vendors.js and has
+ * transport/iframe defined.
+ * @param {string} vendor The vendor name that should be listed in vendors.js
+ */
+function assertVendor(vendor) {
+  user().assert(ANALYTICS_CONFIG &&
+      ANALYTICS_CONFIG[vendor] !== undefined &&
```

Comment:
Check for truthiness instead of `!== undefined`, I think.

Reply:
Fixed.